### ISSUE TITLE
Handling multiple tags in pagination collection

### ISSUE THREAD

**Issue by saenglert** [11ty/eleventy#708]
Hello everyone,
I'm trying to work with Pagination and I would like to pass a collection of data that contains multiple tags.

Example:
```
pagination:
    data: collections.news
```
Takes every item with the tag `new`.

What I would like to do is something akin to:
```
pagination:
    data: collections.getByTagNames("news", "domestic")
```
Which would return all items that have both the `news` and `domestic` tag.

I know that I can add custom collections via the config or add a custom filter. #608

However, both solutions are somewhat impractical to me:
 * Custom Collection: If another tag gets added I need to also add a new collection that includes this tag and also add every possible combination collection of  the tags for filtering.
* Custom Filter: As `data` is a string I can't make function calls at this point in frontmatter and if I use the filter later in the template pagination is off because the pagination result and filter result possibly don't match any more. 

I've noticed #397 proposed (to my understanding) exactly this earlier this year. Has anything come of that?

**Comment by zachleat** [comment]
We can close this out because #711 was merged, yeah?

**Comment by zachleat** [comment]
Closing, this will be available in 0.10.0-beta.2
